src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1900,7 +1910,9 @@ private static Binder GetLambdaEnclosingBinder(int position, CSharpSyntaxNode st
         /// </summary>
         protected void EnsureNullabilityAnalysisPerformedIfNecessary()
         {
-            bool isNullableAnalysisEnabled = IsNullableAnalysisEnabled();
+#pragma warning disable RSEXPERIMENTAL001 // internal use of experimental API
+            bool isNullableAnalysisEnabled = !DisableNullableAnalysis && IsNullableAnalysisEnabled();
```

Comment:
>DisableNullableAnalysis [](http://example.com/codeflow?start=46&length=23)

It looks like ```IsNullableAnalysisEnabled``` already checks ```DisableNullableAnalysis```. Is there a specific reason to check it here explicitly?

Reply:
I think checking `DisableNullableAnalysis` here is redundant. Removing.